frontend/lib/util/clang-integration.cpp

```diff
@@ -54,6 +54,7 @@ namespace util {
 const std::vector<std::string>& clangFlags(Context* context) {
   QUERY_BEGIN_INPUT(clangFlags, context);
   std::vector<std::string> ret;
+  ret.push_back("clang"); // dummy argv[0]
```

Comment:
Why is this necessary -- are we ever calling `clangFlags` and assuming there is something in it before it's been set? I see we `setClangFlags` from the output of `computeClangArgs` which with this PR is guaranteed to have the `clang` executable name as the 0th element already.

Reply:
Yes, we call clangFlags without ever calling setClangFlags in some of the dyno C++ tests. So, it is useful to have something plausible for that case. I will add a comment that it's only expected to be used this way in tests.